articles/cognitive-services/Emotion/QuickStarts/Python.md

```diff
@@ -18,107 +17,143 @@ ms.author: anroth
-    response = conn.getresponse()
-    data = response.read()
-    # 'data' contains the JSON data. The following formats the JSON data for display.
-    parsed = json.loads(data)
-    print ("Response:")
-    print (json.dumps(parsed, sort_keys=True, indent=2))
-    conn.close()
-except Exception as e:
-    print("[Errno {0}] {1}".format(e.errno, e.strerror))
-
-####################################
-
-########### Python 3.2 #############
-import http.client, urllib.request, urllib.parse, urllib.error, base64, sys
-
-headers = {
-    # Request headers. Replace the placeholder key below with your subscription key.
-    'Content-Type': 'application/json',
-    'Ocp-Apim-Subscription-Key': '13hc77781f7e4b19b5fcdd72a8df7156',
-}
-
-params = urllib.parse.urlencode({
-})
-
-# Replace the example URL below with the URL of the image you want to analyze.
-body = "{ 'url': 'http://example.com/picture.jpg' }"
-
-try:
-    # NOTE: You must use the same region in your REST call as you used to obtain your subscription keys.
-    #   For example, if you obtained your subscription keys from westcentralus, replace "westus" in the 
-    #   URL below with "westcentralus".
-    conn = http.client.HTTPSConnection('westus.api.cognitive.microsoft.com')
-    conn.request("POST", "/emotion/v1.0/recognize?%s" % params, body, headers)
-    response = conn.getresponse()
-    data = response.read()
-    # 'data' contains the JSON data. The following formats the JSON data for display.
-    parsed = json.loads(data)
-    print ("Response:")
-    print (json.dumps(parsed, sort_keys=True, indent=2))
-    conn.close()
-except Exception as e:
-    print(e.args)
-####################################
+image_path = "images/emotion_1.jpg"
+image_data = open(image_path, "rb").read()
 ```
 
-## Recognize Emotions Sample Response
-A successful call returns an array of face entries and their associated emotion scores, ranked by face rectangle size in descending order. An empty response indicates that no faces were detected. An emotion entry contains the following fields:
-* faceRectangle - Rectangle location of face in the image.
-* scores - Emotion scores for each face in the image. 
-
-```json
-application/json 
-[
-  {
-    "faceRectangle": {
-      "left": 68,
-      "top": 97,
-      "width": 64,
-      "height": 97
-    },
-    "scores": {
-      "anger": 0.00300731952,
-      "contempt": 5.14648448E-08,
-      "disgust": 9.180124E-06,
-      "fear": 0.0001912825,
-      "happiness": 0.9875571,
-      "neutral": 0.0009861537,
-      "sadness": 1.889955E-05,
-      "surprise": 0.008229999
-    }
-  }
-]
 
+```python
+import requests
+headers  = {'Ocp-Apim-Subscription-Key': subscription_key, "Content-Type": "application/octet-stream" }
+response = requests.post(emotion_recognition_url, headers=headers, data=image_data)
+response.raise_for_status()
+analysis = response.json()
+analysis
+```
+
+
+
+
+    [{'faceRectangle': {'height': 162, 'left': 130, 'top': 141, 'width': 162},
```

Comment:
Expected to list the output here?

Reply:
It looked very small and informative so I kept it. Please let me know if you would like me to remove it.